docs/default_csi_plugin.md

```diff
@@ -0,0 +1,80 @@
+# MicroShift Default CSI Plugin
+
+MicroShift enables dynamic storage provisioning out of the box with the ODF-LVM CSI plugin. This plugin is downstream
+Red Hat distribution of TopoLVM. This feature is not currently configurable but will be in the near future. For more
+information on ODF-LVM, visit the repo's [README](https://github.com/red-hat-storage/topolvm).
+
+## Design
+
+ODF-LVM manifests can be found in [microshift/assets/components/odf-lvm](../assets/components/odf-lvm). For an overview
+of ODF-LVM architecture, see the [design doc](https://github.com/red-hat-storage/topolvm/blob/main/docs/design.md).
+
+## Deployment
+
+ODF-LVM is deployed onto the cluster in the `openshift-storage` namespace, after MicroShift boots. In order to reduce
+MicroShift's storage footprint and cluster overhead,
+[StorageCapacity tracking](https://kubernetes.io/docs/concepts/storage/storage-capacity/) is used in lieu of the
+plugin's extended-scheduler.
+
+## System Requirements
+
+### Volume Group Name
+
+LVMD, a component of ODF-LVM, requires that volume groups specified in
+the [device-class configuration](../assets/components/odf-lvm/topolvm-lvmd-config_configmap_v1.yaml) exist at the
+runtime. If the volume group doesn't exist, the LVMD container will fail to start and enter a CrashLoopBackoff state.
+The initial integration of ODF-LVM assumes volume-group named `rhel` exists. Unallocated space must exist on the volume
+group if the plugin is to dynamically provision logical volumes on it.
```

Comment:
I think these last few sentences can be combined a bit to bring the specific information earlier. Yes, we require that the VG in the device-class exist, but further the name must be `rhel` right now and we should lead with that. When it's configurable, then we can talk about what the default might be (if we have a default), but for now we need to make it clear that MicroShift must be deployed in such a way that there is a VG with that exact name.